packages/core/src/browser/preferences/preference-service.ts

```diff
@@ -459,20 +467,20 @@ export class PreferenceServiceImpl implements PreferenceService {
         return Number(value);
     }
 
-    inspect<T>(preferenceName: string, resourceUri?: string): PreferenceInspection<T> | undefined {
-        const defaultValue = this.inspectInScope<T>(preferenceName, PreferenceScope.Default, resourceUri);
-        const globalValue = this.inspectInScope<T>(preferenceName, PreferenceScope.User, resourceUri);
-        const workspaceValue = this.inspectInScope<T>(preferenceName, PreferenceScope.Workspace, resourceUri);
-        const workspaceFolderValue = this.inspectInScope<T>(preferenceName, PreferenceScope.Folder, resourceUri);
+    inspect<T>(preferenceName: string, resourceUri?: string, forceLanguageOverride?: boolean): PreferenceInspection<T> | undefined {
```

Comment:
The name `forceLanguageOverride` confused me a bit at first. What it means is "don't return the general value if the key is language specific", right? I haven't really thought of a good name, though.

Reply:
I could reverse the polarity and rename it something like `acceptDefault`. There's no exactly antonym for `language overridden`, though...